src/modules/auth/components/auth-view.jsx

```diff
@@ -8,7 +9,7 @@ import AirbitzLogoIcon from 'modules/common/components/airbitz-logo-icon';
 
 import ComponentNav from 'modules/common/components/component-nav';
 
-import { AUTH_SIGNUP, AUTH_LOGIN, AUTH_IMPORT } from 'modules/app/constants/views';
+import { SIGNUP, LOGIN, IMPORT } from 'modules/app/constants/views';
```

Comment:
Nitpick: "sign in/out" and "log in/out" are both used throughout the code.  It would be good to pick one of these and use it.  (The middleware uses "log in/out", FWIW.)